### PR TITLE
Added Parameters section and converted Usage to Examples

### LINKED ISSUE #490: sql: Clean up RENAME TABLE
For the sake of consistency, add a **Parameters** section and convert **Usage** to **Examples**.

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #490

### CODE REVIEW

**Comment by jseldess** [comment]
*[rename-table.md, line 34 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/1532#-Km7DLSzbK-ToHV2Idmh:-Km7DLSzbK-ToHV2Idmi:bu639i) ([raw file](https://github.com/cockroachdb/docs/blob/638cfd978bdca9429cc8909bcd93dd614777f943/rename-table.md#L34)):*
> ```Markdown
> ### Rename a table
> 
> To rename a table, use the `ALTER TABLE` statement followed by the current table name in `database.table` format, the `RENAME TO` statement, and the new table name in `database.table` format:
> ```

As discussed, we probably don't need this introductory text.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/1532)*

File: generate/main.go [hunk 1]
```diff
@@ -611,7 +611,7 @@ type stmtSpec struct {
 	match, exclude []*regexp.Regexp
 	unlink         []string
 	relink         map[string]string
-	nosplit        bool
```
Comment (jseldess) [inline generate/main.go]: You accidentally remove this line, @Amruta-Ranade. It has to be put back in.

**Comment by jseldess** [comment]
@Amruta-Ranade, I just pushed a fix to `main.go` and also regenerated the rename table diagram. The command you used didn't seem to regenerate it properly. We can look into this together more later. 

Once all tests pass, if the rendered page looks good, I'll go ahead and merge. Thanks for this PR!

**Comment by Amruta-Ranade** [comment]
I just regenerated the diagram as well. The rendered page is updated with the new diagram now.

File: generate/main.go [hunk 2]
```diff
@@ -611,7 +611,7 @@ type stmtSpec struct {
...
+    nosplit		   bool
```
Comment (jseldess) [inline generate/main.go]: nit: Can you add spacing to align this variable with the ones above?

**Pull request merged**